docs/spec/staking/transactions.md

```diff
@@ -38,275 +17,150 @@ type TxCreateValidator struct {
     PubKey         crypto.PubKey
     Delegation     sdk.Coin
 }
+```
 
-createValidator(tx TxCreateValidator):
-    ok := validatorExists(tx.ValidatorAddr)
-    if ok return err // only one validator per address
-
-    ok := validatorByPubKeyExists(tx.PubKey)
-    if ok return err // only one validator per public key
-
-    err := validateDenom(tx.Delegation.Denom)
-    if err != nil return err // denomination must be valid
-
-    validator := NewValidator(tx.ValidatorAddr, tx.PubKey, tx.Description)
-
-    err := setInitialCommission(validator, tx.Commission, blockTime)
-    if err != nil return err // must be able to set initial commission correctly
-
-    // set the validator and public key
-    setValidator(validator)
-    setValidatorByPubKeyIndex(validator)
+This message is expected to fail if: 
 
-    // delegate coins from tx.DelegatorAddr to the validator
-    err := delegate(tx.DelegatorAddr, tx.Delegation, validator)
-    if err != nil return err // must be able to set delegation correctly
+ - another validator with this operator address is already registered
+ - another validator with this pubkey is already registered
+ - the initial self-delegation tokens are of a denom not specified as the
+   bonding denom 
+ - the commission parameters are faulty, namely:
+   - `MaxRate` is either > 1 or < 0 
+   - the initial `Rate` is either negative or > `MaxRate`
+   - the initial `MaxChangeRate` is either negative or > `MaxRate`
+ - the description fields are too large
+ 
+This message creates and stores the `Validator` object at appropriate
```

Comment:
Maybe talk about keeper calls, or go into more detail here?

Reply:
nah the keeper should not be specified here. I'm going to add a link at the end of this sentence pointing to the `state.md` however - that should adequately direct those seeking more detail